Tests/Engine/RuleSuppression.tests.ps1

```diff
@@ -106,7 +106,7 @@ function SuppressPwdParam()
     }
 
     Context "Rule suppression within DSC Configuration definition" {
-        It "Suppresses rule" -skip:((Test-PSEditionCoreCLRLinux) -or ($PSVersionTable.PSVersion -lt [Version]'5.0.0')) {
+        It "Suppresses rule" -skip:((!$IsWindows) -or ($PSVersionTable.PSVersion -lt [Version]'5.0.0')) {
```

Comment:
`$PSVersionTable.PSVersion.Major -lt 5` looks more elegant to me but this might be just a question of style and therefore opinionated.

Reply:
that's fine - I have no preference